gamestonk_terminal/behavioural_analysis/README.md

```diff
@@ -165,34 +165,51 @@ Print up to the last 30 messages of a user. [Source: Stocktwits]
+ weighted composite score' is accurate.
+```
+A sample tweet is shown below along with what the VADER polarity scores are.
+
+```'
+Have some $SENS  🚀🏆\n👉FDA approval coming very soon, 👉News is out, amazing test results. \n👉Very high short
+interest rate. \n👉Major short squeeze coming. \n\n$amc $gme $nakd $ctrm $tsla $aapl $oeg $aht $bbby $bb $fsr $spce
+ https://t.co/B0cPhufpA2'
+```
+
+This gives the following scores:
+```python
+created_at                                  Thu Jun 03 20:37:54
+text          Have some $SENS  🚀🏆\n👉FDA approval coming very...
+sentiment                                                 0.872
+positive                                                  0.177
+negative                                                      0
+neutral                                                   0.823
+```
+
+Note that both functions use twitters v2 endpoint, which only gives 7 days of historical data.  It also requires at least 10 tweets be pulled.
 
 ## infer <a name="infer"></a>
 ```
 usage: infer [-n N_NUM]
 ```
-Print quick sentiment inference from last tweets that contain the ticker. This model splits the text into character-level tokens and uses the DistilBERT model to make predictions. DistilBERT is a distilled version of the powerful BERT transformer model. Not only time period of these, but also frequency. Inspired by https://towardsdatascience.com/sentiment-analysis-for-stock-price-prediction-in-python-bed40c65d178. [Source: Twitter]
+Print quick sentiment inference from last tweets that contain the ticker. This model splits the text into character-level tokens and uses theVADER model to make predictions.
   * -n : num of latest tweets to infer from. Default 100.
 
-<img width="948" alt="Captura de ecrã 2021-02-22, às 00 18 22" src="https://user-images.githubusercontent.com/25267873/108643679-808abf80-74a3-11eb-9b50-899be0a4799f.png">
 
 ## sentiment <a name="sentiment"></a>
 ```
 usage: sentiment [-n N_NUM] [-d N_DAYS_PAST]
 ```
-Plot in-depth sentiment extracted from tweets from last days that contain pre-defined ticker. This model splits the text into character-level tokens and uses the DistilBERT model to make predictions. DistilBERT is a distilled version of the powerful BERT transformer model. Note that a big num of tweets extracted per hour in conjunction with a high number of days in the past, will make the algorithm take a long period of time to estimate sentiment. Inspired by https://towardsdatascience.com/sentiment-analysis-for-stock-price-prediction-in-python-bed40c65d178. [Source: Twitter]
-  * -n : num of tweets to extract per hour. Default 100.
-  * -d : num of days in the past to extract tweets. Default 7.
-
-<img width="949" alt="Captura de ecrã 2021-02-22, às 00 15 20" src="https://user-images.githubusercontent.com/25267873/108643678-7ff22900-74a3-11eb-9bea-54f84ffe42e1.png">
-
-![nvda](https://user-images.githubusercontent.com/25267873/108643604-2ab61780-74a3-11eb-950b-ba0bbec8293e.png)
```

Comment:
Why where these plots removed?

The change of the NLP model used shouldn't really influence the plot. Am I right? If so, would be nice to keep them.

Reply:
Well I redid the plots, so while they should be similar, there are some differences I figured you would want them updated.

Reply:
Got it! With the Neutral? I'll test this at some point!

Nicely done :)